### ISSUE TITLE
Add Localized __content.js for Page-Level Overall Styling

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#75]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```

It's quite difficult to ask users to consolidate all the pages' specific Javascript files into assets/ directory. We can offer `__content.js` as a direct source instead.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
`__content.js` will be loaded as the last CSS list item governing the page styling.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
`__content.js` is not responsive to hestiaHUGO.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
implemented in b62e1f4638239c7f34b92fc48491f0717635259f

**Comment by hollowaykeanho** [comment]
reworded in a23192e8b483635186af1ca51a46b11ef3e9fc5a.

**Comment by hollowaykeanho** [comment]
Released in `v1.1.0`.
